assets/app/views/browse/route.html

```diff
@@ -111,24 +111,44 @@
                     <dd ng-if-end>{{route.spec.tls.insecureEdgeTerminationPolicy || 'None'}}</dd>
                     <dt>Certificate:</dt>
                     <dd>
-                      <span ng-if="route.spec.tls.certificate" click-to-reveal><pre class="clipped">{{route.spec.tls.certificate}}</pre></span>
+                      <span ng-show="route.spec.tls.certificate && !reveal.certificate">
+                        <a href="" ng-click="reveal.certificate = true">Show</a>
```

Comment:
nope not needed, just making a mental note that this works :)

On Fri, Mar 18, 2016 at 8:58 AM, Sam Padgett <notifications@github.com>
wrote:

> In assets/app/views/browse/route.html
> <https://github.com/openshift/origin/pull/8108#discussion_r56651472>:
>
> > @@ -111,24 +111,44 @@
> >                      <dd ng-if-end>{{route.spec.tls.insecureEdgeTerminationPolicy || 'None'}}</dd>
> >                      <dt>Certificate:</dt>
> >                      <dd>
> > -                      <span ng-if="route.spec.tls.certificate" click-to-reveal><pre class="clipped">{{route.spec.tls.certificate}}</pre></span>
> > +                      <span ng-show="route.spec.tls.certificate && !reveal.certificate">
> > +                        <a href="" ng-click="reveal.certificate = true">Show</a>
>
> It's not necessary, but I can add it if you prefer.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8108/files/3cb8eceeeb3198b2498ba8b48eff0e318eb7c1d4#r56651472>
>